README.md

```diff
@@ -127,8 +103,38 @@ Check out `./cockroach help` to see all available commands.
 ## Deploying CockroachDB in production
 
 To run a CockroachDB cluster on various cloud platforms using [docker machine](http://docs.docker.com/machine/),
-see [cockroach-prod](https://github.com/cockroachdb/cockroach-prod)
+see [cockroach-prod](https://github.com/cockroachdb/cockroach-prod).
+
+We'll set up a three-node cluster below.
```

Comment:
that's a good point, we should also avoid "deploying". `Running a multi-node cluster` should do the trick for now.